modelkit/assets/cli.py

```diff
@@ -24,41 +26,34 @@ def assets_cli():
     pass
 
 
-gcs_fn_re = r"gs://(?P<bucket_name>[\w\-]+)/(?P<object_name>.+)"
+storage_url_re = (
+    r"(?P<storage_prefix>[\w]*)://(?P<bucket_name>[\w\-]+)/(?P<object_name>.+)"
+)
 
 
-def parse_gcs(path):
-    match = re.match(gcs_fn_re, path)
+def parse_remote_url(path):
+    match = re.match(storage_url_re, path)
     if not match:
-        raise ValueError(f"Could not parse GCS path `{path}`")
+        raise ValueError(f"Could not parse path `{path}`")
     return match.groupdict()
 
 
-def _download_object_or_prefix(manager, asset_path, destination_dir):
-    parsed_path = parse_gcs(asset_path)
+def _download_object_or_prefix(driver, object_name, destination_dir):
     asset_path = os.path.join(destination_dir, "myasset")
```

Comment:
NB: I don't understand the need to use "myasset" subdir. I have the impression we could simplify here by downloading directly in destination dir.

Reply:
myasset is not a subdir, it is a temporary asset name

Reply:
actually, it is a subdir (or prefix) when "object_name" is a directory.
anyway, I see no reason to override "object_name" in the temporary directory.
it is unrelated to your change so I can deal with that in another PR.